### PR TITLE
Fix/#71

### PR DESCRIPTION
**Pull request by saemideluxe**

### CODE REVIEW

File: basxconnect/core/layouts/settings_layout.py [hunk 1]
```diff
@@ -101,6 +101,7 @@ def generate_term_datatable(title, category_slug):
             "add",
             query={
                 "category": cat.id,
+                "next": reverse("basxconnect.core.views.personsettings"),
```
Comment (wipascal) [inline basxconnect/core/layouts/settings_layout.py:104]: I read this function with a very general name and docstring and then I am surprised to see that is is hardcoded for `personsettings`. I would maybe update either function name or docstring to reflect the fact that this is only for `personsettings`.

Comment (saemideluxe) [reply basxconnect/core/layouts/settings_layout.py:104]: Right, we have the following options for improvment:

1. Naming and documenting the function specific for the settings view
2. Refactor to let ```"next"``` and ```backurl``` be taken from a parameter to the function (maybe a single argument is enough)
3. Change the behaviour to always return to the current page.

I think a combination of 2. and 3. would be nice, with 3 beeing the default behaviour. But for that we need to add support for the the urls to be lazy elements (in the Datatables element). This is currently not true but would make a lot of sense.

Comment (wipascal) [reply basxconnect/core/layouts/settings_layout.py:104]: If it's too much work for now, you can also add 2. and 3. as "todo" comments in the code, I am fine with that.

Comment (saemideluxe) [reply basxconnect/core/layouts/settings_layout.py:106]: Okay, I have quite a load this and next week, so I just adjusted the docstring and add a TODO

**Pull request merged**